main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
It seems like 8 stories are being created at the moment,  if we update this line to  `(loop < 11)`
we'll be creating 10 stories as stated in the requirements :)